pep-0999.rst

```diff
@@ -0,0 +1,175 @@
+PEP: 999
```

Comment:
Is there a specific reason to use 999? I think we can use the next available number: 536 :)